docs/source/anesthetic.read.rst

```diff
@@ -40,6 +40,7 @@ anesthetic.read.hdf module
 .. automodule:: anesthetic.read.hdf
    :members:
    :undoc-members:
+   :show-inheritance:
```

Comment:
This line needs to go, otherwise it will cause autodoc to try and find documentation in parents that do not have any docs...